src/types.js

```diff
@@ -36,6 +36,22 @@ export function isArray(value) {
 }
 
 /**
+ * Converts an array-like object to an array.
+ * @param {?HTMLCollection|?NodeList|*} arrayLike
```

Comment:
For reference, [Closure's doc comment](https://github.com/google/closure-library/blob/master/closure/goog/array/array.js#L809-L818).